src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -27,5 +28,11 @@ public EventHubAttribute(string eventHubName)
         /// </summary>
         [AutoResolve]
         public string EventHubName { get; private set; }
+
+        /// <summary>
+        /// Optional connection name. If missing, tries to use a registered event hub sender.
+        /// </summary>
+        [AppSetting]
```

Comment:
'Correct' behavior would be [AppSetting] to use build in resolution - all the other built ins use custom nameResolution code, which could likely be converted to AppSetting but is out of scope here